packages/devel/glibc/package.mk

```diff
@@ -144,6 +144,11 @@ post_makeinstall_target() {
   if [ "$TARGET_ARCH" = "arm" -a "$TARGET_FLOAT" = "hard" ]; then
     ln -sf ld.so $INSTALL/usr/lib/ld-linux.so.3
   fi
+
+# add cross ldd script
+  mkdir -p $TOOLCHAIN/bin/
+    cp $PKG_DIR/scripts/cross-compile-ldd $TOOLCHAIN/bin/$TARGET_NAME-ldd
+    chmod +x $TOOLCHAIN/bin/$TARGET_NAME-ldd
```

Comment:
Wouldn't it be more useful (and easier to use/remember) if a constant generic name such as  `cross-compile-ldd` is used when copying to `$TOOLCHAIN/bin`,  instead of the `$TARGET_NAME` which varies from project to project? You could easily embed the tuple information inside the script so that you don't need to extract this from the script filename.

Reply:
I'll look at the script more thoroughly later today and send you a patch for this PR that should allow the tuple to be embedded altering the script to be cashed anything we want (of course the installed script will only work for the toolchain in which it is installed but I don't really see that as a problem).

Reply:
what's the point though?

Reply:
Ease of use? Oh forget it.